src/core/public/chrome/ui/header/header_help_menu.tsx

```diff
@@ -35,14 +35,20 @@ import {
   EuiHorizontalRule,
 } from '@elastic/eui';
 
-import { ExclusiveUnion } from '@elastic/eui';
-import { combineLatest } from 'rxjs';
-import { HeaderExtension } from './header_extension';
-import { ChromeHelpExtension } from '../../chrome_service';
+import { InternalApplicationStart } from '../../../application';
 import { GITHUB_CREATE_ISSUE_LINK, KIBANA_FEEDBACK_LINK } from '../../constants';
+import { ChromeHelpExtension } from '../../chrome_service';
+import { HeaderExtension } from './header_extension';
+import { isModifiedOrPrevented } from './nav_link';
+
+/** @public */
+export type ChromeHelpExtensionLinkBase = Pick<
+  EuiButtonEmptyProps,
+  'iconType' | 'target' | 'rel' | 'data-test-subj'
+>;
```

Comment:
Restrict the links base type (which was `EuiButtonEmptyProps` previously) to the only properties that make sense for the helpMenu links.